src/Test/UnitTests/attributes.jl

```diff
@@ -31,6 +31,27 @@ function silent(model::MOI.ModelLike, config::TestConfig)
 end
 unittests["silent"] = silent
 
+"""
+    timelimit(model::MOI.ModelLike, config::TestConfig)
+
+Test that the [`MOI.TimeLimit`](@ref) attribute is implemented for `model`.
+"""
+function timelimit(model::MOI.ModelLike, config::TestConfig)
+    if config.solve
+        @test MOI.supports(model, MOI.TimeLimit())
+        # Get the current value to restore it at the end of the test
+        value = MOI.get(model, MOI.TimeLimit())
+        MOI.set(model, MOI.TimeLimit(), value + 1)
+        @test value + 1 == MOI.get(model, MOI.TimeLimit())
```

Comment:
Use `isapprox` instead of `==`. The solver might store the time in a `Float32` for instance and won't be able to return exactly the same value

Reply:
I wasn't sure because I had issues with isapprox(. , 0). I'm changing that.

Reply:
You need to set `atol` to a nonzero value for `isapprox` to work when one of the argument is zero

Reply:
See https://github.com/JuliaLang/julia/issues/23376

Reply:
That what I did for me but in a test I have no clue how to choose the right `atol`

Reply:
I choosed `atol = eps(0.0)`. Testing locally now.